diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -141,12 +141,13 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
         }
     }
 
+    @Suppress("ComplexMethod")
     private fun handleOperatorWithLineBreakBefore(node: ASTNode) {
         if (node.isDotFromPackageOrImport()) {
             return
         }
         val isIncorrect = node.run {
-            if (isCallsChain()) {
+            if (isCallsChain() && containsManyFunctions()) {
```

Comment:
Please add fixme somewhere that we can't distinguish fully qualified names from chain of calls for now (using only AST), but should do it in the future.